index.html

```diff
@@ -0,0 +1,359 @@
+              <div class="col-12">
+                <p>
+                  Tion and dislike men who are so beguiled and demoralized by
+                  desire by the charms of pleasure of the moment, so blinded by
+                  desire, that they cannot foresee Ddislike men who are so
+                  beguiled and demoralized by the charms of pleasure of the
+                  moment, so blinded by desire, that.
+                </p>
+              </div>
+              <div class="col-12 WhyChooseUs_button">
+                <a href="#" class="button">Explore Places </a>
+              </div>
+            </div>
+          </div>
+          <div class="col-6 col-lg-6 col-sm-12 morph">
+            <div class="row">
+              <div class="col-lg-12 col-sm-6 first">
+                <img class="first" src="img/Image2.png" alt="house" />
+              </div>
+              <div class="col-lg-12 col-sm-6 second">
+                <img class="second" src="img/Image1.png" alt="mountains" />
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="SearchPlaces">
+      <div class="container">
+        <div class="row">
+          <div class="col-4">
+            <div class="row">
+              <div class="col-12"><span class="icon-search"></span></div>
+              <div class="col-12"><p>Search Places</p></div>
+            </div>
+          </div>
+          <div class="col-4">
+            <div class="row">
+              <div class="col-12"><span class="icon-check-circle"></span></div>
+              <div class="col-12"><p>Select Places</p></div>
+            </div>
+          </div>
+          <div class="col-4">
+            <div class="row">
+              <div class="col-12"><span class="icon-thumbs-up"></span></div>
+              <div class="col-12"><p>Book & Enjoy</p></div>
+            </div>
+          </div>
+          <object
+            class="dotted_line"
+            data="img/Line.svg"
+            type="image/svg+xml"
+          ></object>
+        </div>
+      </div>
+    </section>
+    <section class="firstSpain">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-7 col-xl-6 col-sm-12 firstSpain_block">
+            <div class="row">
+              <div class="col-12"><p>Spain</p></div>
+              <div class="col-12"><h3>Grand Spain Madrid</h3></div>
+              <div class="col-12">
+                <p class="firstSpain_p">
+                  But I must explain to you how all this mistaken idea of
+                  account to denouncing pleasure and praising pain was account
+                  born and I account will give you a complete account of the
+                  system
+                </p>
+              </div>
+              <div class="col-12 firstSpain_button">
+                <a href="#" class="btn-1">book now</a>
+              </div>
+            </div>
```

Comment:
1. Незачем всё заворачивать в **col-12**. 
2. Такая же карточка используется в следующей секции, поэтому лучше выделить её в отдельный блок. Вместо `.firstSpain_block` и ` .secondSpain_block` будет` .card`
```suggestion
            <div class="card">
              <p class="card__subtitle">Spain</p>
              <h3 class="card__title">Grand Spain Madrid</h3>
              <p class="card__text">
                  But I must explain to you how all this mistaken idea of
                  account to denouncing pleasure and praising pain was account
                  born and I account will give you a complete account of the
                  system
                </p>
                <a href="#" class="button card__button">book now</a>
            </div>
```